core/src/main/java/io/grpc/internal/DelayedClientTransport.java

```diff
@@ -367,9 +367,10 @@ void endBackoff() {
     }
   }
 
+  // TODO(carl-mastrangelo): remove this once the Subchannel change is in.
   @Override
-  public String getLogId() {
-    return GrpcUtil.getLogId(this);
+  public LogId getLogId() {
+    return LogId.allocate(getClass().getName());
```

Comment:
Even if this is temporary, we should not allocate a new ID every single time `getLogId()` is called.